src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/client/manageusers/ManageUsersClient.kt

```diff
@@ -16,6 +17,7 @@ class ManageUsersClient(@Qualifier("manageUsersWebClient") private val webClient
         .uri("/users/{username}", username)
         .retrieve()
         .bodyToMono(UserDetailsDto::class.java)
+        .retryNetworkExceptions()
```

Comment:
Is there some way we can add this as an extension to WebClient if we're wanting it generally? I'd just be worried someone will forget it

Reply:
Related to above comment, it does need to be an optional include cause we can't guarantee all downstream endpoints are idempotent